eng/ApiListing.exclude-attributes.txt

```diff
@@ -3,6 +3,7 @@ T:System.Runtime.CompilerServices.AsyncStateMachineAttribute
 T:System.Runtime.CompilerServices.CompilerGeneratedAttribute
 T:System.Runtime.CompilerServices.NullableContextAttribute
 T:System.Runtime.CompilerServices.NullableAttribute
+T:System.Runtime.CompilerServices.IsReadOnlyAttribute
```

Comment:
Attribute is used by the compiler to mark an auto-property.   It is marked `internal` in the `netstandard2.0` implementation and comments say that it should be applied only by the compiler.   _([net8.0 docs](https://learn.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.isreadonlyattribute?view=net-8.0&viewFallbackFrom=netstandard-2.0))_

This attribute does not get added when converting an auto property to an explicit getter implementation.   As there is no visibility nor impact to the developer experience using the public API, there's no need to track with API Compat.